assets/airports/rjbb.json

```diff
@@ -0,0 +1,2732 @@
+{
+    "radio": {
+        "twr": "Kansai Tower",
+        "app": "Kansai Approach",
+        "dep": "Kansai Radar"
+    },
+    "icao": "RJBB",
+    "iata": "KIX",
+    "magnetic_north": -7,
+    "ctr_radius": 80,
+    "ctr_ceiling": 7000,
+    "initial_alt": 3000,
+    "position": ["N34d26m03.00", "N135d13m58.00"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N34d26m03.00", "N135d13m58.00"],
+    "has_terrain": false,
+    "wind": {
+		"angle": 204,
+        "speed": 9
+    },
+    "airspace": [{
+        "floor": 0,
+        "ceiling": 20000,
+        "airspace_class": "C",
+        "poly": [
+			["N33d35m04.00", "E134d59m50.00"],
+			["N33d35m56.00", "E134d40m54.00"],
+			["N34d17m01.00", "E133d57m30.00"],
+			["N34d47m42.00", "E135d00m09.00"],
+			["N34d56m49.00", "E135d48m57.00"],
+			["N34d58m51.00", "E135d57m45.00"],
+			["N34d39m14.00", "E136d05m25.00"],
+			["N34d39m14.00", "E136d05m25.00"],
+			["N34d16m09.00", "E136d09m43.00"],
+			["N33d35m04.00", "E134d59m50.00"]
+        ]
+    }],
+    "fixes": {
+		"AKASI": ["N34d35m00.30",	"E134d45m00.30"],
+		"ALISA": ["N34d30m42.60",	"E134d08m04.10"],
+		"ALLAN": ["N34d14m04.10",	"E134d57m28.30"],
+		"ASAMI": ["N34d07m57.90",	"E134d48m41.80"],
```

Comment:
the fixes section is an example where hard tabs are present and should be converted.